src/asset/asset-registry.js

```diff
@@ -352,8 +344,9 @@ pc.extend(pc, function () {
 
                     self.fire("load", asset);
                     self.fire("load:" + asset.id, asset);
-                    if (asset.file && asset.file.url) {
-                        self.fire("load:url:" + asset.file.url, asset);
+                    var assetFileUrl = asset.getFileUrl();
+                    if (assetFileUrl) {
+                        self.fire("load:url:" + assetFileUrl, asset);
```

Comment:
Not too sure about which URL to use here, the asset.file.url or the full URL with the prefix.